src/actions/package/providers/yay.rs

```diff
@@ -0,0 +1,95 @@
+use super::PackageProvider;
+use crate::actions::{package::PackageVariant, ActionError};
+use crate::utils::command::{run_command, Command};
+use serde::{Deserialize, Serialize};
+use std::process::{Command, Output, Stdio};
```

Comment:
This conflicts with `Command` above. I don't believe we need this import anymore. It can be removed.